.test-infra/jenkins/dependency_check/generate_report.sh

```diff
@@ -20,6 +20,9 @@
 set -e
 set -v
 
+# Get currently used Python version from args or assume a default.
+PYTHON=${1:-python3}
```

Comment:
Does this work if users are using an alias/symlink?

Reply:
yes, it should work. python3 is typically a symlink.

Reply:
alias probably wouldn't work

Reply:
verified that symlinks work and aliases don't